### PR TITLE
Add size comparison for MDC-based list

### PR DESCRIPTION
**Pull request by devversion**
Adds the size comparison for the MDC-based list in the current state. The current results are not really representative as, we usethe full MDC checkbox instead of having fine-grained base CSS from it (we have a proposal for the MDC team).

At first glance, more JS comes from the fact that we have a few more classes. i.e. the base classes and separate class for the action list (not the case in the non-MDC one). Also we have the typical low-hanging fruit / CSS overhead from MDC CSS variables. Also CSS for dividers and multi lines needs to be replicated due to MDC limitations...

We can use this setup as a starting point though to monitor size / determine issues. We'd need to set up the comparison anyway at some point, so why not yet.

### CODE REVIEW

File: projects/mat-list/.browserslistrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```
Comment (mmalerba) [inline projects/mat-list/.browserslistrc:1]: can delete this file

File: projects/mat-mdc-list/.browserslistrc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
+# For additional information regarding the format and rule options, please see:
```
Comment (mmalerba) [inline projects/mat-mdc-list/.browserslistrc:2]: can delete this file as well

File: projects/mat-list/.browserslistrc (hunk 1 above)
Comment (devversion) [reply projects/mat-list/.browserslistrc:1]: good catch. I forgot staging the deletions (the collect script deletes them)

**Comment by devversion** [comment]
@mmalerba Addressed feedback. Should be good for merge.

**Pull request merged**